doc/src/index.md

```diff
@@ -127,19 +127,25 @@ language. In addition to the above, some advantages of Julia over comparable sys
   * Powerful shell-like capabilities for managing other processes
   * Lisp-like macros and other metaprogramming facilities
 
-## [Julia Core, Base and the Standard Library](@id man-core-base-and-stdlib)
+## [Julia Standard Modules and the Standard Library](@id man-standard-modules-stdlib)
+
+The Julia runtime comes with [standard modules](@ref standard-modules),
+which are essential namespaces that are usually loaded automatically.
 
 ```@docs; canonical=false
 Core
 Base
```

Comment:
I assume it's deliberate, but just to be sure: why don't you include `Main` here too?

Reply:
My thinking was that `Main` is not really a namespace where people should be looking for either Julia builtin symbols or "standard library" packages/modules, because the symbols under `Main` are user-defined. So, to keep this section as a reference for "what do I get out of the box", I omitted it. I don't feel very strongly about it, but in some ways I didn't like the aesthetic of just duplicating the Standard Modules section from `base.md`.

Reply:
Fair enough!